.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -11,34 +11,46 @@ jobs:
       win_64_cuda_compilercuda-nvcccuda_compiler_version12.5python3.10.____cpython:
         CONFIG: win_64_cuda_compilercuda-nvcccuda_compiler_version12.5python3.10.____cpython
         UPLOAD_PACKAGES: 'True'
+        SHORT_CONFIG: win_64_cuda_compilercuda-nvcccuda_c_ha04e48f122
       win_64_cuda_compilercuda-nvcccuda_compiler_version12.5python3.11.____cpython:
         CONFIG: win_64_cuda_compilercuda-nvcccuda_compiler_version12.5python3.11.____cpython
         UPLOAD_PACKAGES: 'True'
+        SHORT_CONFIG: win_64_cuda_compilercuda-nvcccuda_c_ha3b2c00765
       win_64_cuda_compilercuda-nvcccuda_compiler_version12.5python3.12.____cpython:
         CONFIG: win_64_cuda_compilercuda-nvcccuda_compiler_version12.5python3.12.____cpython
         UPLOAD_PACKAGES: 'True'
+        SHORT_CONFIG: win_64_cuda_compilercuda-nvcccuda_c_h55c0af3228
       win_64_cuda_compilercuda-nvcccuda_compiler_version12.5python3.9.____cpython:
         CONFIG: win_64_cuda_compilercuda-nvcccuda_compiler_version12.5python3.9.____cpython
         UPLOAD_PACKAGES: 'True'
+        SHORT_CONFIG: win_64_cuda_compilercuda-nvcccuda_c_h0a1d50dd92
       win_64_cuda_compilernvcccuda_compiler_version11.8python3.10.____cpython:
         CONFIG: win_64_cuda_compilernvcccuda_compiler_version11.8python3.10.____cpython
         UPLOAD_PACKAGES: 'True'
+        SHORT_CONFIG: win_64_cuda_compilernvcccuda_compil_h200ec4340e
       win_64_cuda_compilernvcccuda_compiler_version11.8python3.11.____cpython:
         CONFIG: win_64_cuda_compilernvcccuda_compiler_version11.8python3.11.____cpython
         UPLOAD_PACKAGES: 'True'
+        SHORT_CONFIG: win_64_cuda_compilernvcccuda_compil_haec70b9094
       win_64_cuda_compilernvcccuda_compiler_version11.8python3.12.____cpython:
         CONFIG: win_64_cuda_compilernvcccuda_compiler_version11.8python3.12.____cpython
         UPLOAD_PACKAGES: 'True'
+        SHORT_CONFIG: win_64_cuda_compilernvcccuda_compil_h3457fb9941
       win_64_cuda_compilernvcccuda_compiler_version11.8python3.9.____cpython:
         CONFIG: win_64_cuda_compilernvcccuda_compiler_version11.8python3.9.____cpython
         UPLOAD_PACKAGES: 'True'
+        SHORT_CONFIG: win_64_cuda_compilernvcccuda_compil_hf10085d904
   timeoutInMinutes: 360
   variables:
     CONDA_BLD_PATH: D:\\bld\\
     SET_PAGEFILE: 'True'
     UPLOAD_TEMP: D:\\tmp
 
   steps:
+    - powershell: |
+        reg add "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock" /t REG_DWORD /f /v "AllowDevelopmentWithoutDevLicense" /d "1"
+      displayName: Enable Developer Mode on Windows
+      # https://learn.microsoft.com/en-us/windows/apps/get-started/developer-mode-features-and-debugging
```

Comment:
As using `mklink` on Windows, which `git` needs for symlinks`, requires Developer Mode. This enables that

https://learn.microsoft.com/en-us/windows/apps/get-started/developer-mode-features-and-debugging#use-powershell-to-enable-your-device